vendor/golang.org/x/crypto/go.mod

```diff
@@ -1,9 +1,11 @@
 module golang.org/x/crypto
 
-go 1.11
+go 1.17
 
 require (
-	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110
-	golang.org/x/sys v0.0.0-20201119102817-f84b799fce68
+	golang.org/x/net v0.0.0-20211112202133-69e39bad7dc2
+	golang.org/x/sys v0.0.0-20210615035016-665e8c7367d1
```

Comment:
didn't have to update this one, as we're already ahead https://github.com/golang/sys/compare/665e8c7367d1...69cdffdb9359ff97d91e4f4fbb6b2714c3898eae